modules/tasks/tasks/src/main/java/org/motechproject/tasks/domain/mds/task/TaskActionInformation.java

```diff
@@ -204,6 +204,7 @@ public boolean equals(Object obj) {
 
         return Objects.equals(this.serviceInterface, other.serviceInterface) &&
                 Objects.equals(this.serviceMethod, other.serviceMethod) &&
+                Objects.equals(this.specifiedName, other.specifiedName) &&
```

Comment:
please add this field in hashCode and Equals methods to DataSource, TaskACtionInformationDto, DataSourceDto